### YamlMime:ManagedReference
items:
- uid: System.Security.XmlSyntaxException
  id: XmlSyntaxException
  children:
  - System.Security.XmlSyntaxException.#ctor
  - System.Security.XmlSyntaxException.#ctor(System.Int32)
  - System.Security.XmlSyntaxException.#ctor(System.String)
  - System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  - System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException
  fullName: System.Security.XmlSyntaxException
  type: Class
  summary: "Wyjątek zgłaszany, gdy występuje błąd składni w analiza kodu XML. Ta klasa nie może być dziedziczona."
  remarks: "Xmlsyntaxexception — używa domyślnej <xref:System.Object.Equals%2A>wdrażania, która obsługuje równości odwołań.</xref:System.Object.Equals%2A>       Listę początkowe wartości właściwości dla wystąpienia xmlsyntaxexception — Zobacz konstruktora."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public sealed class XmlSyntaxException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor
  id: '#ctor'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klasy przy użyciu domyślnej właściwości."
  remarks: "W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Nieprawidłowa składnia. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException ();
    parameters: []
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  id: '#ctor(System.Int32)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klasy z numer wiersza, w których wykryto wyjątek."
  remarks: "Numer wiersza w strumieniu XML jest aktualizowany w każdym przypadku, gdy zostanie wykryta znaku nowego wiersza. Numerowanie rozpoczyna się od 1 do pierwszego wiersza.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Nieprawidłowa składnia w wierszach `lineNumber`. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "Numer wiersza w strumieniu XML, na których wykryto błąd składni XML."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> klasy z powodu określonego błędu."
  remarks: "W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  id: '#ctor(System.Int32,System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> z określonego komunikatu o błędzie i numer wiersza, w których wykryto wyjątek."
  remarks: "Numer wiersza w strumieniu XML jest aktualizowany w każdym przypadku, gdy zostanie wykryta znaku nowego wiersza. Numerowanie rozpoczyna się od 1 do pierwszego wiersza.       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; Nieprawidłowa składnia w wierszach `lineNumber`  -  `message` &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber, string message);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "Numer wiersza w strumieniu XML, na których wykryto błąd składni XML."
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "Inicjuje nowe wystąpienie klasy <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> z określonego komunikatu o błędzie i odwołania do wyjątek wewnętrzny będący przyczyną tego wyjątku."
  remarks: "Wyjątek zgłaszany bezpośrednio w wyniku poprzedniego wyjątku powinien zawierać odwołanie do poprzedniego wyjątku we <xref:System.Exception.InnerException%2A>Właściwości.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>Właściwość zwraca tę samą wartość, która została przekazana do konstruktora, lub `null` Jeśli <xref:System.Exception.InnerException%2A>Właściwości nie dostarcza do konstruktora wartości wyjątku wewnętrznego.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       W poniższej tabeli przedstawiono początkowe wartości właściwości dla wystąpienia <xref:System.Security.XmlSyntaxException>.</xref:System.Security.XmlSyntaxException>      | Właściwość | Wartość |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Wyjątek wewnętrzny odwołania. |   | <xref:System.Exception.Message%2A>| Ciąg z komunikatem o. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Komunikat o błędzie opisujący przyczynę wyjątek."
    - id: inner
      type: System.Exception
      description: "Wyjątek będący przyczyną bieżącego wyjątku. Jeśli `inner` parametr nie jest <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, bieżący wyjątek jest zgłaszany w <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloku, który obsługuje wyjątek wewnętrzny."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Security.XmlSyntaxException.#ctor
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Security.XmlSyntaxException.#ctor*
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException.XmlSyntaxException
